packages/xstate-wallet/src/index.ts

```diff
@@ -1,20 +1,20 @@
 import {ChannelWallet} from './channel-wallet';
 import {MessagingService} from './messaging';
 import {ChainWatcher} from './chain';
-import {IndexedDBBackend} from './store/indexedDB-backend';
 import {MemoryBackend} from './store/memory-backend';
 import {XstateStore} from './store';
 import * as constants from './constants';
 import Url from 'url-parse';
 import './render';
 
 import {logger} from './logger';
+import {Backend} from './store/dexie-backend';
 const log = logger.info.bind(logger);
 
 (async function() {
   const chain = new ChainWatcher();
 
-  const backend = constants.USE_INDEXED_DB ? new IndexedDBBackend() : new MemoryBackend();
+  const backend = constants.USE_INDEXED_DB ? new Backend() : new MemoryBackend();
```

Comment:
This is unrelated to the PR but I think we should retire `MemoryBackend` soon and just rely on `CLEAR_STORAGE_ON_START` if we want to start fresh on refresh.

Reply:
It's not clear how easy a task that is: https://github.com/statechannels/monorepo/issues/1526